recipes/bwa2/2.0pre2_npg/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "2.0pre2_npg" %}
```

Comment:
Please use `git describe` to get a version number see e.g. https://github.com/kjsanger/npg_conda/tree/conda-recipe-structure/recipes/htslib